<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="baclark77-mac1.home" name="All Concurrent Web Tests" tests="1" failures="1" timestamp="22 Apr 2014 13:38:09 GMT" time="6.874" errors="0">
  <testcase name="CreateCharacterProfile_Test" time="3.988" classname="com.nbcuni.test.publisher.tests.ContentEntityCreationManagement.ContentTypesEntities.CharacterProfile.CreateCharacterProfile">
    <failure type="junit.framework.AssertionFailedError" message="Fail this test...">
      <![CDATA[junit.framework.AssertionFailedError: Fail this test...
	at junit.framework.Assert.fail(Assert.java:47)
	at com.nbcuni.test.publisher.tests.ContentEntityCreationManagement.ContentTypesEntities.CharacterProfile.CreateCharacterProfile.CreateCharacterProfile_Test(CreateCharacterProfile.java:32)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:724)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- CreateCharacterProfile_Test -->
</testsuite> <!-- All Concurrent Web Tests -->
